frontend/integration-tests-cypress/cypress/support/index.d.ts

```diff
@@ -0,0 +1,12 @@
+// load type definitions that come with Cypress module
```

Comment:
We should get these from the `@types/cypress` dependency instead of copying them into the repo.

Reply:
Fixed, replaced `...support/index.d.ts` with `...support/index.ts`
See [comment](https://github.com/openshift/console/pull/4831#issuecomment-611705992)